### PR TITLE
Review chapter "Git, GitHub and Markdown"

### PR DESCRIPTION
**Pull request by ludelafo**

### CODE REVIEW

File: 03-git-github-and-markdown/COURSE_MATERIAL.md [hunk 1]
```diff
@@ -1307,10 +1635,30 @@ Once the pull request is approved, you will receive an invitation to join the
...
+If you want, you can change the visibility of your membership in the GitHub
+organization. You can choose between public and private.
+
+You can do so by going in the settings of the organization and then in the
+**Membership** tab.
+
+This will allow other users to see your membership in the organization. They
+will be able to see your username and the date you joined the organization.
+
+This is totally optional. You can keep your membership private if you want.
```
Comment (ludelafo) [inline 03-git-github-and-markdown/COURSE_MATERIAL.md:1661]: I would need to do some tests with you @hadrylouis to validate this part whenever you are available.

File: 03-git-github-and-markdown/COURSE_MATERIAL.md [hunk 2]
```diff
@@ -817,55 +1012,201 @@ profile README!
...
+
+    Initial commit
+
+```
+
+You can notice that the new commit `2ea9f1c4` is the HEAD of the local `main`
+branch.
+
+The `origin/main` and `origin/HEAD` are still pointing to the `20f87413` commit
+as their HEAD.
```
Comment (hadrylouis) [inline 03-git-github-and-markdown/COURSE_MATERIAL.md:1131]: Is it normal not to see the latest “Update profile README” commit in the `git log`

File: 03-git-github-and-markdown/COURSE_MATERIAL.md [hunk 3]
```diff
@@ -900,7 +1241,8 @@ repository describing what you want to do:
 - **Description**: I would like to add myself to the list of students in the
   GitHub organization README.
 
-> [!TIP]  
+> [!TIP]
+>
 > While totally optional, it is a good to create an issue before starting to
```
Comment (hadrylouis) [inline 03-git-github-and-markdown/COURSE_MATERIAL.md:1246]: typo : `it is a good practice` maybe

**Review by hadrylouis — CHANGES_REQUESTED** [review]

File: 03-git-github-and-markdown/COURSE_MATERIAL.md (hunk 2 above)
Comment (ludelafo) [reply 03-git-github-and-markdown/COURSE_MATERIAL.md]: Fixed!

**Review by hadrylouis — APPROVED** [review]

**Pull request merged**